docs/howto/object-storage/s3/versioning.md

```diff
@@ -53,22 +108,72 @@ the following commands:
     ```bash
     aws --profile <region> \
       --endpoint-url=https://s3-<region>.{{extra.brand_domain}}:8080 \
-      s3api get-object \ 
+      s3api get-object \
       --bucket <bucket-name> \
-	  --key <object-name> \
-	  --version-id <versionid> \
-	  <local-filename>
+      --key <object-name> \
+      --version-id <versionid> \
+      <local-filename>
     ```
 === "mc"
     ```bash
     mc cp \
-	  --version-id <versionid> \
-	  <region>/<bucket-name>/<object-name> \
-	  <local-filename>
+      --version-id <versionid> \
+      <region>/<bucket-name>/<object-name> \
+      <local-filename>
     ```
 === "s3cmd"
     This functionality is not available with the `s3cmd` command.
 
 When you download an object from a versioned bucket *without*
 specifying a version identifier, your S3 client will download the
 latest version of the object that the bucket contains.
+
+
+## Deleting a versioned object
+
+Like the commands to *create* objects, the commands to *delete* them
```

Comment:
I would slightly rephrase: "Like the commands for object creation, the commands for object deletion behave differently"

Reply:
I would disagree. :) When given the choice to describe an action with a verb or with an action noun, the verb is the superior option because it is more concise and direct.